ex5.py

```diff
@@ -0,0 +1,181 @@
+class User:
+    """A User  class. Allows managing users .
+
+       Parameters
+       ----------
+       user_name : str
+           name of the user.
+       password : str
+           password of the user.
+       user_type : str
+           ype of user (manager/regular user).
+       Attributes
+       ----------
+      user_name : str
+           name of the user.
+      password : str
+           password of the user.
+      user_type : str
+           ype of user (manager/regular user).
+       """
+
+    def __init__(self, user_name, password, user_type):
+        self.user_name = user_name
+        self.password = password
+        self.user_type = user_type
+
+    def __str__(self):
+        return f"user name:{self.user_name} user type:{self.user_type}"
+
+
+class File:
+    """A FIle class. Allows creating Basic Files .
+
+           Parameters
+           ----------
+           size : int
+               size of the file.
+           content : str
+               the content of the file.
+           creator : User
+               the creator of the file.
+           Attributes
+           ----------
+           size : int
+                size of the file.
+           content : str
+               the content of the file.
+           creator : User
+               the creator of the file.
+           """
+
+    def __init__(self, size, content, creator):
+        self.size = size
+        self.content = content
+        self.creator = creator
+
+    def __str__(self):
+        return f"file size:{self.size} file content:{self.content} file creator:{self.creator}"
+
+
+class TextFile(File):
```

Comment:
You were right to inherit File's features, but are there no specific variables to be initiated for text files? 
Same goes for binary file.